tests/js/spec/stores/guideStore.spec.jsx

```diff
@@ -78,4 +90,19 @@ describe('GuideStore', function() {
     GuideStore.onFetchSucceeded(data);
     expect(GuideStore.state.currentGuide).not.toEqual(null);
   });
+
+  it('should update', function() {
+    let mockRecordCue = jest.fn();
+    GuideStore.recordCue = mockRecordCue;
+
+    GuideStore.onRegisterAnchor(anchor1);
+    GuideStore.onRegisterAnchor(anchor2);
+    GuideStore.onFetchSucceeded(data);
+    expect(mockRecordCue).toHaveBeenCalledWith(data.issue.id, data.issue.cue);
+    expect(mockRecordCue).toHaveBeenCalledTimes(1);
```

Comment:
this test seems to cover the branches according to codecov but I'm not sure how it does (but could just be missing something). Could you explain with a comment or write another case like `it('should not update on duplicates')`?

I was looking to see (in pseudocode) something like:

triggerguide(A)
expect(mockRecordCue).toHaveBeenCalledTimes(1);
triggerguide(A)
expect(mockRecordCue).toHaveBeenCalledTimes(1);
triggerguide(B)
expect(mockRecordCue).toHaveBeenCalledTimes(2);